src/errors.rs

```diff
@@ -39,16 +34,16 @@ pub enum Error<E>
     ///
     /// Typically this indicates a faulty implementation of `serde::Serialize`.
     ServerSerialize(String),
-    /// The server canceled the response before it was completed.
-    ReplyCanceled,
     /// The server was unable to reply to the rpc for some reason.
+    ///
+    /// This is a service-specific error. It is individually specified in the
+    /// `service!` macro for each rpc.
```

Comment:
> It is individually specified in the `service!` macro for each rpc.

I think it's more accurate to say it's specified in the implementation of the method, rather than specified in the `service!` macro unless I'm missing something.